### PR TITLE
Update README.md

### LINKED ISSUE #5247: Docs: Proofread Getting Started guide for clarity
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Similar to #4861, but for the Getting Started guide. It hasn't been reworked in a little while and I'd like to take an editing pass on it.

### Affected URL(s)

- https://typescript-eslint.io/docs/linting
- https://typescript-eslint.io/docs/linting/*

### PR DESCRIPTION
**Pull request by Zamiell**
- don't use conjunctions
- don't use double-negatives
- clarify what TSLint is

#5247

### CODE REVIEW

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Thanks for sending this! The latter two changes I agree with and find pleasing. 😄

File: docs/linting/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar_label: Linting your TypeScript Codebase
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Whether you're adding linting to a new TypeScript codebase, adding TypeScript to an old codebase, or migrating from the deprecated [TSLint](https://www.npmjs.com/package/tslint), the steps aren't a whole lot different.
+Whether you are adding linting to a new TypeScript codebase, adding TypeScript to an old codebase, or migrating from the deprecated [TSLint](https://www.npmjs.com/package/tslint) tool, the steps to follow are similar.
```
[the remark this answers is not in the document]
Comment (Zamiell) [reply docs/linting/README.md:10]: In my experience, use of contractions in technical writing is minimized. The benefits of conveying a friendly tone have to be weighed against making the text marginally harder for foreign-language speakers to understand. Here, we expect that people throughout the world will be using the TypeScript programming language, so the documentation should reflect that.

[the remark this answers is not in the document]
Comment (Zamiell) [reply docs/linting/README.md:10]: image

[the remark this answers is not in the document]
Comment (Josh-Cena) [reply docs/linting/README.md:10]: "'re" won't be confused with anything; the TS-ESLint docs aren't formal writing either. I do prefer no-contractions myself, but I can't find a reason strong enough to warrant a change, particularly if it sets a precedent.

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Coming back to this PR: the discussion is interesting but the sentence in question is removed by #5248. Once #5248 goes in we can close this PR.

**Comment by JoshuaKGoldberg** [comment]
Pardon my closing the PR - since #5248 is in, this PR is against deleted code. Thanks though! ❤️

**Pull request closed without merging**